src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -293,39 +292,65 @@ private void FillDoubleArrayFromJsArray(double[] array, ArrayInstance jsArray, s
                     ++i;
                 }
             }
-            
-            private JsValue AppendTimeSeries(JsValue self, JsValue[] args)
+
+            private JsValue TimeSeries(JsValue self, JsValue[] args)
```

Comment:
What would this API look like for counters?

Reply:
something like this ? :
`counters(doc).increment(name)`
`counters(doc).increment(name, value)`
`counters(doc).delete(name)`
`counters(doc).get(name)`

and maybe also :
`counters(doc).getAll()`
`counters(doc).getRaw(name)`

Reply:
Looking at this, I don't know if that would be worth our time

Reply:
@ppekrol ?

Reply:
It is nice, but we still probably would want to keep both of the syntaxes for some time. Because of that, I think we can do it later on in 5.x cycle and deprecate old syntax in 6 or 7 even. Thoughts?

Reply:
Make sense, we can delay that, then